Gems/AtomLyIntegration/ImguiAtom/Code/Source/DebugConsole.cpp

```diff
@@ -183,6 +165,25 @@ namespace AZ
         }
     }
 
+    void DebugConsole::AddDebugLog(const char* window, const char* debugLogString, AZ::LogLevel logLevel)
+    {
+        AZ::ILogger* logger = AZ::Interface<AZ::ILogger>::Get();
+        if (logger == nullptr || logLevel < logger->GetLogLevel())
+        {
+            return;
+        }
+
+        AZ::Color color = GetColorForLogLevel(logLevel);
+        if (strcmp(window, AZ::Debug::Trace::GetDefaultSystemWindow()) == 0)
+        {
+            AddDebugLog( debugLogString, color);
+        }
```

Comment:
Can this check be case insensitive?

Reply:
If so, `azstricmp` can replace it.

Reply:
We could use a case insensitive check but it's a compile time constant so it's should be safe to use case sensitive compare.
```
static const char*    g_dbgSystemWnd = "System";
```